app/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    buildToolsVersion '25.0.0'
```

Comment:
I don't think you should update the build tools unless it's necessary for this PR to work.

Reply:
You are right. This was made by a click I made on Android Studio when I first open this project there. There is no need of updating the build tool version for this PR to work \o/